ircb/stores/logs.py

```diff
@@ -36,6 +32,23 @@ def create(cls, hostname, roomname, message, event, timestamp,
         session.commit()
         return log
 
+    @classmethod
+    def get(cls, filter=None, order_by=None, limit=30, sort='timestamp'):
+        qs = session.query(MessageLog)
+        if filter:
+            for key, value in filter.items():
+                qs = qs.filter(getattr(MessageLog, key) == value)
+
+        if order_by:
+            for item in order_by:
+                if item.startswith('-'):
+                    qs = qs.order_by(getattr(MessageLog, item[1:]).desc())
+                else:
+                    qs = qs.order_by(getattr(MessageLog, item))
+
+        qs = qs.limit(limit)
+        return sorted(qs.all(), key=lambda item: getattr(item, 'timestamp'))
```

Comment:
The sorted here seems odd considering the order_by above.

Plus it makes you iterate through all the results while you could ask the DB to do it for you :)